hs_app_netCDF/receivers.py

```diff
@@ -14,15 +14,17 @@
 from hs_core.hydroshare import utils
 from hs_app_netCDF.forms import VariableValidationForm, OriginalCoverageForm, VariableForm
 from hs_app_netCDF.models import NetcdfResource
-import nc_functions.nc_utils as nc_utils
-import nc_functions.nc_dump as nc_dump
-import nc_functions.nc_meta as nc_meta
+import hs_file_types.nc_functions.nc_utils as nc_utils
+import hs_file_types.nc_functions.nc_dump as nc_dump
+import hs_file_types.nc_functions.nc_meta as nc_meta
 
 
 # receiver used to extract metadata after user click on "create resource"
 @receiver(pre_create_resource, sender=NetcdfResource)
 def netcdf_pre_create_resource(sender, **kwargs):
-
+    # Pabitra: Though there is some common code between netcdf file type and here as it relates
```

Comment:
@alvacouch Certainly it is possible. I have kind of done that for Raster file type. This one is more complicated than raster. I can take another look at this prior to merging.